components/layout_2020/positioned.rs

```diff
@@ -919,6 +878,26 @@ impl<'a> AbsoluteAxisSolver<'a> {
         }
     }
 
+    fn solve_tentativelly(&mut self) -> AxisResult {
+        self.solve(None::<fn() -> ContentSizes>)
```

Comment:
Is the type annotation really necessary here?

Reply:
Yes, I added the method to avoid having to type the annotation.

```
error[E0283]: type annotations needed
   --> components/layout_2020/positioned.rs:871:20
    |
871 |         self.solve(None)
    |              ----- ^^^^ cannot infer type of the type parameter `T` declared on the enum `Option`
    |              |
    |              required by a bound introduced by this call
    |
    = note: multiple `impl`s satisfying `_: FnOnce()` found in the following crates: `alloc`, `core`:
            - impl<A, F> FnOnce<A> for &F
              where A: std::marker::Tuple, F: Fn<A>, F: ?Sized;
            - impl<A, F> FnOnce<A> for &mut F
              where A: std::marker::Tuple, F: FnMut<A>, F: ?Sized;
            - impl<Args, F, A> FnOnce<Args> for std::boxed::Box<F, A>
              where Args: std::marker::Tuple, F: FnOnce<Args>, A: Allocator, F: ?Sized;
            - impl<F, Args> FnOnce<Args> for Exclusive<F>
              where F: FnOnce<Args>, Args: std::marker::Tuple;
            - impl<R, F> FnOnce<()> for AssertUnwindSafe<F>
              where <F as FnOnce<()>>::Output == R, F: FnOnce();
note: required by a bound in `AbsoluteAxisSolver::<'a>::solve`
   --> components/layout_2020/positioned.rs:785:51
    |
785 |     fn solve(&self, get_content_size: Option<impl FnOnce() -> ContentSizes>) -> AxisResult {
    |                                                   ^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `AbsoluteAxisSolver::<'a>::solve`
help: consider specifying the generic argument
    |
871 |         self.solve(None::<T>)
    |                        +++++
```